tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -236,7 +236,113 @@ public function test_state_not_printed_when_empty() {
 		$this->assertEquals( array( 'config' => array( 'myPlugin' => array( 'a' => 1 ) ) ), $result );
 	}
 
+	/**
+	 * Tests that empty state objects are pruned from printed data.
+	 *
+	 * @ticket 60761
+	 *
+	 * @covers ::print_client_interactivity_data
+	 */
+	public function test_state_not_printed_when_empty_array() {
+		$this->interactivity->state( 'pluginWithEmptyState_prune', array() );
+		$this->interactivity->state( 'pluginWithState_include', array( 'value' => 'excellent' ) );
+		$printed_script = get_echo( array( $this->interactivity, 'print_client_interactivity_data' ) );
+		$expected       = <<<'SCRIPT_TAG'
+<script type="application/json" id="wp-interactivity-data">
+{"state":{"pluginWithState_include":{"value":"excellent"}}}
+</script>
+
+SCRIPT_TAG;
+
+		$this->assertEquals( $expected, $printed_script );
```

Comment:
We usually prefer `assertSame` over `assertEquals` whenever possible. Can you update the new tests accordingly?

Reply:
Done.